rust/tests/test.bats

```diff
@@ -0,0 +1,5 @@
+@test "Version matches plan" {
+  TEST_PKG_VERSION="$(echo "${TEST_PKG_IDENT}" | cut -d/ -f3)"
+  actual_version="$(hab pkg exec "${TEST_PKG_IDENT}" rustc --version | awk '{print $2}')"
+  [[ "$actual_version" = "${TEST_PKG_VERSION}" ]]
```

Comment:
`[[` should be avoided

See https://github.com/sstephenson/bats/issues/49 for context

Reply:
I think there's other pitfalls with `[[` other than that BATs warning; I'm not familiar enough with bash to cite any though...

Reply:
Hi @james-stocks.  I've changed the check to use a diff instead.